### DISCUSSION TITLE
Community Meeting 2023 Aug 18

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#364] [Announcements]
Topics:

- Workflow for amending a PR by a contributor without write access to repo (see #357 and #361 for ilustration)
    - See https://github.com/pi-base/data/wiki/Reviewing
- Threshold to invite frequent contributors to be reviewers with push access to repo
- Ambiguity examples:
    - https://topology.pi-base.org/spaces/S000147 vs https://topology.pi-base.org/spaces/S000148
    - https://topology.pi-base.org/spaces/S000082
- When "immediate from definitions", what references are needed?
    - see at least https://github.com/pi-base/data/pull/407
- Best way to handle "converse" of theorem when the full list needed for justification would be very long
    - see e.g. https://github.com/pi-base/data/pull/409#issuecomment-1683301933

**Comment by ccaruvana** [comment]
Not sure if this is the most relevant place for this comment, but I was wondering if it would be of interest to look into implementing a feature in the search that identifies spaces that don't have a T/F value for a given property. E.g. searching "?Semiregular" could return the list of spaces in the pi-base for which the pi-base doesn't know whether P10 holds or not.

**Reply by prabau** [reply]
Sounds like it could be useful for knowing what would possibly need work.  To add to the wish list for the next web interface.

**Reply by StevenClontz** [reply]
I want to say that this used to be a feature, many years ago.

**Reply by ccaruvana** [reply]
Posted for future reference at https://github.com/pi-base/web/issues/70

**Comment by StevenClontz** [comment]
I'm propose Aug 18 for this next meeting (modulo possible conflicts for carolinas seminar)

**Comment by StevenClontz** [comment]
Reminder of this Friday's call (I don't think Carolina seminar is meeting)

**Comment by StevenClontz** [comment]
See everyone at https://southalabama.zoom.us/j/91451774321 at 3pm Central
